e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/wiremock/service/ExternalServiceEMTest.java

```diff
@@ -0,0 +1,120 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ExternalServiceEMTest extends SpringTestBase {
+    /**
+     * Test to check the AdditionalInfoDto transaction between driver and core.
+     *
+     * ExecutionTracer reset the AdditionalInfoList several times during the start-up. As the
+     * result, calls made during the start-up got captured and get cleared during this.
+     *
+     * TODO: Test will be disabled till ExecutionTracer reset issues handled
+     */
+
+    private static WireMockServer wireMockServer;
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+
+        DnsCacheManipulator.setDnsCache("foo.bar", "127.0.0.2");
+        DnsCacheManipulator.setDnsCache("baz.bar", "127.0.0.2");
+
+        wireMockServer = new WireMockServer(new WireMockConfiguration().bindAddress("127.0.0.2").port(8080).extensions(new ResponseTemplateTransformer(false)));
+        wireMockServer.start();
+
+        /*
+         * WireMock endpoint will respond the third value of the request path
+         * as JSON response.
+         * */
+        wireMockServer.stubFor(get(urlMatching("/api/echo/([a-z]*)"))
+                .atPriority(1)
+                .willReturn(
+                        aResponse()
+                                .withHeader("Content-Type", "application/json")
+                                .withStatus(200)
+                                .withBody("{\"message\": \"{{request.path.[2]}}\"}")
+                                .withTransformers("response-template")));
+
+        wireMockServer.stubFor(get(urlMatching("/api/echo/([a-z]*)\\?x=([0-9]*)&y=([a-z]*)"))
+                .atPriority(1)
+                .willReturn(
+                        aResponse()
+                                .withHeader("Content-Type", "application/json")
+                                .withStatus(200)
+                                .withBody("{\"message\": \"{{request.path.[2]}}\"}")
+                                .withTransformers("response-template")));
+
+        // to prevent from the 404 when no matching stub below stub is added
+        wireMockServer.stubFor(get(urlMatching("/.*"))
+                .atPriority(2)
+                .willReturn(aResponse()
+                        .withStatus(200)
+                        .withBody("Not found!!")));
+
+        ServiceController serviceController = new ServiceController();
+        EMConfig config = new EMConfig();
+        config.setInstrumentMR_NET(true);
+        SpringTestBase.initClass(serviceController,config);
+    }
+
+    @AfterAll
+    public static void shutdownServer() {
+        wireMockServer.stop();
+        DnsCacheManipulator.clearDnsCache();
+    }
+
+    @Disabled
```

Comment:
if you have `@Disabled` tests, add comment why

Reply:
I've added the comment why it's disabled under the class and removed the line comments.